frontend/src/components/Forms/ShoutoutsPage/ShoutoutCard.tsx

```diff
@@ -24,7 +43,20 @@ const ShoutoutCard = (props: {
         <ShoutoutDeleteModal uuid={shoutout.uuid} setGivenShoutouts={setGivenShoutouts} />
       </Card.Group>
       <Card.Content description={shoutout.message} />
+
+      {isLoading ? (
+        <Loader active inline />
+      ) : (
+        image && (
```

Comment:
Maybe also check to see if the image string is non-empty and non-null like image?.length > 0 since it could be possible that an empty string or a malformed image URL might still pass the condition. Might need clarification on this.